doc/Administration-guide/System-installation/Cluster-setup/index.md

```diff
@@ -10,55 +10,53 @@ sort_idx: 30
+```
 
-In this guide we assume that you had already set up your SQL and NoSQL database clusters, so this tutorial doesn't cover such themes like setting up ones for Cassandra, MongoDB or PostgreSQL. 
-Refer to official [Cassandra](http://docs.datastax.com/en/landing_page/doc/landing_page/current.html), [MongoDB](https://docs.mongodb.com/manual/) and [PostgreSQL](https://www.postgresql.org/docs/) documentation in order to setup corresponding database cluster.
+To refresh the key files, stop the kaa-node service, replace the key files, and start the service again.
 
-MariaDB come with out-of-the-box master-master replication support thus we recommend to use this database in your Kaa cluster. 
-You can find detailed instructions in [MariaDB cluster setup guide]({{root_url}}Administration-guide/System-installation/Cluster-setup/MariaDB-cluster-setup-guide/).
+The Kaa node service requires some third party dependencies, such as SQL and NoSQL databases, and Apache Zookeeper service.
+For more information, see [Architecture overview]({{root_url}}Architecture-overview/).
+A choice of databases (for example, MongoDB + PostgreSQL) depends on your particular use case.
+For database cluster installation, refer to the official [Cassandra](http://docs.datastax.com/en/landing_page/doc/landing_page/current.html), [MongoDB](https://docs.mongodb.com/manual/) and [PostgreSQL](https://www.postgresql.org/docs/) documentation.
 
-> Note: In a cluster you need to connect to databases from external hosts so you need to allow such external connections in corresponding database security configurations and configure firewall rules for database host machine. 
-> Refer to official documentation for corresponding database for security configuration details.
+MariaDB comes with the master-master replication support out of the box, therefore it is recommended that you use this database in your Kaa cluster.
+See [MariaDB cluster setup guide]({{root_url}}Administration-guide/System-installation/Cluster-setup/MariaDB-cluster-setup-guide/).
 
-In addition to added on ```kaa-node``` service installation step firewall rules on every endpoint Kaa Administrator need to add some more rules for databases (depending on a set databases in the cluster) and Zookeeper ports.
+To allow connections to the databases from external hosts, configure the corresponding database security settings and set up the firewall rules for the database host machine.
+For security configuration settings, refer to the database official documentation.
 
-> Note: This configuration will not affect AWS deployment or any other cloud provider and must be applied only if you setting up cluster on VMs or separate Linux machines. 
-> Next instructions depend on specific cloud provider. 
+In addition to the firewall rules set up during the Kaa node service installation process, you need to add some rules for the databases (depending on the choice of databases in the cluster) and Zookeeper ports.
```

Comment:
Please add:
... you need to add some rules for the databases (depending on the choice of databases in the cluster), Kaa node thrift and Zookeeper ports.